src/logging/monitor.ts

```diff
@@ -236,6 +234,7 @@ class LogMonitor {
 		this.lastSentTimestamp = timestamp;
 	}
 
+	// eslint-disable-next-line @typescript-eslint/require-await
 	private async handleHostServiceRow(
```

Comment:
Since the function doesn't really await the systemHook, the `async` is not needed so we can probably remove it